### ISSUE TITLE
Problem with CRTM installation on Orion

### ISSUE THREAD

**Issue by CatherineThomas-NOAA** [NOAA-EMC/hpc-stack#527]
**Describe the bug**
CI testing for the global-workflow on Orion results in this error message in the log file (as seen in [g-w PR 1818](https://github.com/NOAA-EMC/global-workflow/pull/1818)):

> 73: Apply_AntCorr(FAILURE) : Input iFOV inconsistent with AC data

These errors are from a problem with the CRTM installation. As pointed out by @emilyhcliu, the MHS Metop-C coefficient file is incorrect; it is too small. It only contains header information:
```
Orion-login-4[4] cthomas$ pwd
/apps/contrib/NCEP/libs/hpc-stack/intel-2018.4/crtm/2.4.0/fix
Orion-login-4[5] cthomas$ ls -l mhs*metop-?.*
-rw-r----- 1 8536 noaa-hpc 11396 Feb  4  2022 mhs_metop-a.SpcCoeff.bin
-rw-r----- 1 8536 noaa-hpc 49212 Feb  4  2022 mhs_metop-a.TauCoeff.bin
-rw-r----- 1 8536 noaa-hpc 11396 Feb  4  2022 mhs_metop-b.SpcCoeff.bin
-rw-r----- 1 8536 noaa-hpc 49212 Feb  4  2022 mhs_metop-b.TauCoeff.bin
-rw-r----- 1 8536 noaa-hpc   492 Feb  4  2022 mhs_metop-c.SpcCoeff.bin
-rw-r----- 1 8536 noaa-hpc 49212 Feb  4  2022 mhs_metop-c.TauCoeff.bin
```

**Expected behavior**
The MHS Metop-C files should be the same size as Metop-A and Metop-B.

**System:**
Orion

**Comment by aerorahul** [comment]
@Hang-Lei-NOAA Is this resolvable in short order?

**Comment by Hang-Lei-NOAA** [comment]
Yes, this is resolvable. The fix files in CRTM/2.4.0 were recently updated by GSI team. But this installation is the old one.
It is better to use : EPIC installation on orion, which has been updated.
module use /work/noaa/epic-ps/hpc-stack/libs/intel/2022.1.2/modulefiles/stack
